Woof/Woof/NetworkServiceStub.swift

```diff
@@ -0,0 +1,9 @@
+import NetworkService
+
+/// Stub network service to check the integration of `NetworkService` package.
+struct NetworkServiceStub {
```

Comment:
Why do we need these struct compiled into the build app?
If you want to test it, you should create a test for this and integrate package within the test case.

Reply:
Ok, I got the idea, I will create a test to verify the integration of the package instead of adding a dummy file that should be removed in the future.

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/commit/ecc37e88f92e7110ab5ffd3a38e860c1731972b0